_data/sites.json

```
@@ -1501,6 +1501,16 @@
         ]
     },
 
+    {
+        "name": "Cointree",
+        "url": "https://cointree.formstack.com/forms/account_closure_request",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "hard",
```
I'd label this as `hard`, because it requires involvement with the customer support.

---

@CharlesZ54 does it? From the wording, it's comparable to a normal delete-your-account form, but the account isn't instantly deleted.

---

There's no involvement on the user's end. Once the form is submitted, the account is queued to be closed by their support team.

---

Hmm, then it's ok to be medium, thanks for clarifying!